db/hardware_test.go

```diff
@@ -0,0 +1,241 @@
+// Package db - hardware tests
+// The following tests validate database functionality, using both network and
+// instance payloads. A postgres instance with the tink database schema is
+// required in order to run the following tests.
+package db
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"sync"
+	"testing"
+
+	"github.com/google/uuid"
+
+	_ "github.com/lib/pq"
+)
+
```

Comment:
Can you change the test names so its pseudo-namespaced please, TestHardware*, TestTemplate*, TestWorkflow*, ...

Reply:
nm, lets do this in a follow up PR.

Reply:
nm lets do this in a follow up and avoid more large diffs here.